packages/react-core/src/components/MultipleFileUpload/examples/MultipleFileUploadBasic.tsx

```diff
@@ -101,50 +98,54 @@ export const MultipleFileUploadBasic: React.FunctionComponent = () => {
   const successfullyReadFileCount = readFileData.filter(fileData => fileData.loadResult === 'success').length;
 
   return (
-    <MultipleFileUpload
-      onFileDrop={handleFileDrop}
-      dropzoneProps={{
-        accept: 'image/jpeg, application/msword, application/pdf, image/png',
-        onDropRejected: handleDropRejected
-      }}
-    >
-      <MultipleFileUploadMain>
-        <MultipleFileUploadTitle>
-          <MultipleFileUploadTitleIcon />
-          <MultipleFileUploadTitleText>
-            Drag and drop files here
-            <MultipleFileUploadTitleTextSeparator>or</MultipleFileUploadTitleTextSeparator>
-          </MultipleFileUploadTitleText>
-        </MultipleFileUploadTitle>
-        <MultipleFileUploadButton />
-        <MultipleFileUploadInfo>Accepted file types: JPEG, Doc, PDF, PNG</MultipleFileUploadInfo>
-      </MultipleFileUploadMain>
-      {showStatus && (
-        <MultipleFileUploadStatus
-          statusToggleText={`${successfullyReadFileCount} of ${currentFiles.length} files uploaded`}
-          statusToggleIcon={getStatusIcon()}
-        >
-          {currentFiles.map(file => (
-            <MultipleFileUploadStatusItem
-              file={file}
-              key={file.name}
-              onClearClick={() => removeFiles([file.name])}
-              onReadSuccess={handleReadSuccess}
-              onReadFail={handleReadFail}
-            />
-          ))}
-        </MultipleFileUploadStatus>
-      )}
-      <Modal
-        isOpen={!!modalText}
-        title="Unsupported file"
-        titleIconVariant="warning"
-        showClose
-        aria-label="unsupported file upload attempted"
-        onClose={() => setModalText('')}
+    <>
+      <Checkbox
```

Comment:
I definitely can, but I believe that having the checkbox at the top is cleaner as otherwise the checkbox will be pushed down when each file is added and I believe that makes for a less than great experience.

Additionally part of my reason for having the checkbox at the top was that it is done in [other](https://www.patternfly.org/v4/components/card/#modifiers) [components](https://www.patternfly.org/v4/components/file-upload/#custom-file-upload) as well, primarily in situations like this where the toggle will move if placed below the example.

Reply:
Good call out.  I did not notice the other one.  I do think we should be consistent across components.  I would think we should change the other one as well. @mcarrano do you have any thoughts in this?

Reply:
Not sure I understand the question @tlabaj . What other one?

Reply:
@mcarrano The checkbox to toggle `isHorizontal` is at the top.  We usually put them at the bottom.  Austin pointed out that the  single file upload component also has an example with the checkbox at the top.  I think we should be consistent across out components and have them at the bottom.  So I thin the example here should change ant the one in the single file upload as well.

Reply:
@tlabaj I'll move the checkbox here, but there are other instances of checkboxes being at the top (for instance in [card](https://www.patternfly.org/v4/components/card/#modifiers)) scattered throughout the codebase, and since we're already at planned code freeze day + 1 I would rather hunting them all down and fixing them to one standard be done under a separate issue.

Thoughts?

Reply:
@wise-king-sullyman good to know we can open an issue to make it consistent across the code base.